src/dune

```diff
@@ -1,4 +1,4 @@
 (library
  (name tiff)
  (public_name tiff)
- (libraries fmt cstruct optint))
+ (libraries fmt eio cstruct optint))
```

Comment:
Is this change necessary? I can't see where you're relying on eio yet.

Reply:
Woops! That was a mistake, thanks.